src/common/lib/api/rating.js

```diff
@@ -1,4 +1,9 @@
-import apiClientRequest from "@/common/lib/api";
+import apiClientRequest from "@/common/lib/api"
+
+const auth = {
```

Comment:
i thought this auth part already done by @bukandicki ?

Reply:
Oh sorry , I missed to revert this .